config/element_io_develop.json

```diff
@@ -0,0 +1,13 @@
+{
+  "posthog": {
+    "api_key": "phc_MhClVy9DiV20vazSYIiedFkM5Xi3z1LPBwrdn9PYZQQ",
+    "api_host": "https://app.posthog.com"
```

Comment:
Is that the right place? I thought we were using a self-hosted Posthog

Reply:
Good point - @toger5 ?

Reply:
No not yet. We are using a seperate instance for call. And a second self hosted one would cost a lot more.